cli/tools/test/mod.rs

```diff
@@ -547,6 +573,17 @@ pub async fn run_tests_for_worker(
     used_only,
   }))?;
   let mut had_uncaught_error = false;
+  let stats = worker.js_runtime.runtime_activity_stats_factory();
+  let ops = worker.js_runtime.op_names();
+  let op_id_host_recv_message = ops
+    .iter()
+    .position(|op| *op == "op_host_recv_message")
+    .unwrap();
+  let op_id_host_recv_ctrl = ops
+    .iter()
+    .position(|op| *op == "op_host_recv_ctrl")
+    .unwrap();
```

Comment:
Why are these special cased?

Reply:
These are a special case for the existing op sanitization as they are "weird" ops that may appear to leak. These will be used in the follow-up PR.